content/docs/how-we-work/task-estimation/_index.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Task Estimation
+weight: 14
+bookCollapseSection: true
+description: In our Agile projects, estimates define the work size needed to meet acceptance criteria or project milestones. We use story points, not hours, to estimate effort, considering complexity, work volume, and risk. This emphasizes that story points are a measure of relative effort, not actual time. Using story points helps us improve estimation accuracy and project delivery.
+---
+
+# Task Estimation
+
+In our Agile projects, estimating the work needed to meet acceptance criteria or project milestones is essential for effective planning and delivery. We use story points instead of hours to estimate effort, considering factors like complexity, work volume, and risk. This approach emphasizes that story points measure relative effort rather than actual time, helping us improve estimation accuracy and project delivery.
```

Comment:
@swarad07 Are we using SP across all projects? In CDM we are not anymore using it as we are in Scrumban (Scrum + Kanban) and providing estimate in hours.

In case if SP are used in Staff aug, I think we can make this generic and not mention anything about hours or SP. What do you think?

Reply:
@Kalaiselvan88 @Yemaneberhan-Lemma In the Confluence page, we wanted to capture everything related to Estimations. This page is also linked to many client onboardings, and content has been vetted by clients as well.

We don't use SP in all managed and staff projects either, but it sees to be the recommended way in most AGILE frameworks and the documentation makes it clear that while SP is encouraged, it is alright to use hours. The page captures the nuances well.

> The estimations in AGILE projects are typically done using Story points, although it is not uncommon to use hours. 

> No matter how you estimate, use hours, or story points, or which technique is used. Task breakdown is the key, the more smaller your work, the better your estimate and the smaller the error in case you are wrong.

As for the page in the Eng Handbook (in this PR), I agree it is not quite capturing this essence and seems to lean more towards SP. We should change the line to something like this,

`We can also use story points instead of hours to estimate effort, considering factors like complexity, work volume, and risk. This approach emphasizes that story points measure relative effort rather than actual time, helping us improve estimation accuracy and project delivery. However, this can vary from project to project and there are scenarios where hours might be preferred over Story points.`

Reply:
ok sounds good @swarad07 

@Yemaneberhan-Lemma @hussainweb I recommended above with understanding that we want to write the current practices we do in baseline rather than the ideal or recommended approach. I am ok with adding both SP and hours if we are ok with suggesting both.